pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>org.ohdsi</groupId>
     <artifactId>circe</artifactId>
-    <version>1.3.1-SNAPSHOT</version>
+    <version>1.3.2-SNAPSHOT</version>
```

Comment:
Per semver semantics, I think this should be updated to 1.4.0-SNAPSHOT:  we aren't doing a bugfix, this is a new feature which doesn't break existing functionality (at least based on this: https://semver.org/)

If you agree, can you update the version to 1.4.0-SNAPSHOT?